src/pages/ChooseArea.svelte

```diff
@@ -142,40 +151,52 @@
   }
 </script>
 
-<div class="left">
-  <h1>Welcome to ATIP v2</h1>
-  <button type="button" on:click={() => (showAbout = !showAbout)}>About</button>
-
-  <p>Select Transport Authority or Local Authority District:</p>
-  <div>
-    <input
-      data-testid="transport-authority"
-      list="authorities-list"
-      bind:value={inputValue}
+<div class="govuk-grid-row">
```

Comment:
Using https://design-system.service.gov.uk/styles/layout/ to split the page into two halves